src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.Packages.cs

```diff
@@ -89,12 +89,12 @@ public OutdatedPackageException(string message, PackageInfo package, SemanticVer
 
             if (package.IsRoslyn)
             {
-                var buildVersion = package.Version.GetSuffixBuildVersion();
+                var packageBuildVersion = package.Version.GetSuffixBuildVersion();
 
-                if (buildVersion.Build != roslynBuildVersion.FiveDigitBuildNumber ||
-                    buildVersion.Revision != roslynBuildVersion.Revision)
+                if (packageBuildVersion.Build != buildVersion.FiveDigitBuildNumber ||
+                    packageBuildVersion.Revision != buildVersion.Revision)
                 {
-                    throw new InvalidOperationException($"Roslyn package version '{package.Version}' inconsistent with build version '{roslynBuildVersion}'");
+                    throw new InvalidOperationException($"Roslyn package version '{package.Version}' inconsistent with build version '{buildVersion}'");
```

Comment:
Does the "Roslyn" in there need to be dropped too?

Reply:
its not the "Roslyn" version.  It the project system/LUT etc build version.

Reply:
Exactly, shouldn't the word "Roslyn" be removed then?

Reply:
Ah yes. sorry, thought you were talking about the variable change from `roslynBuildVersion` -> `buildVersion` not the message text.